src/Microsoft.DotNet.Build.Tasks/PackageFiles/toolruntime.targets

```diff
@@ -13,7 +13,6 @@
 
     <ToolTargetFramework Condition="'$(ToolTargetFramework)' == ''">DNXCore,Version=v5.0</ToolTargetFramework>
     <ToolArchitecture Condition="'$(ToolArchitecture)' == ''">x64</ToolArchitecture>
```

Comment:
I believe that both ToolTargetFramework and ToolArchitecture shouldn't be needed anymore either. I'm not 100% sure though, but if we don't then perhaps we can clean that out as well.